Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -333,4 +333,20 @@ getCurrentLocationMarker :: String -> String
 getCurrentLocationMarker currentVersion = if isPreviousVersion currentVersion (getPreviousVersion "") then "ic_customer_current_location" else "ny_ic_customer_current_location"
 
 getPreviousVersion :: String -> String 
-getPreviousVersion _ = if os == "IOS" then "1.2.5" else "1.2.0"
\ No newline at end of file
+getPreviousVersion _ = if os == "IOS" then "1.2.5" else "1.2.0"
+
+rotateArray :: forall a. Array a -> Int -> Array a
```

Comment:
@Vignesh-772  Why is this function required?

Reply:
It is used to rotate the array. As currently we are directly passing the FareBreakUp from backend it has total fare in 0th index so I need to move that to last element and It might useful in some other cases so I put this function in `Utils.purs`